kapitan/refs/base.py

```diff
@@ -355,13 +355,15 @@ def __init__(self, content, content_type):
 
 
 class RefController(object):
-    def __init__(self, path):
+    def __init__(self, path,target_name = '',inventory_path = './inventory'):
```

Comment:
I had another look at this PR and compared with the GPG implementation for secrets. Actually you don't need to add `target_name` here at all. 
If you look at `kapitan/kapitan/refs/secrets/gpg.py` in the line `target_name = ref_params.kwargs['target_name']` it's actually getting the target_name from the inventory since it already exists there. 
You could easily do the same in your vaultkv class and get it that way instead of adding this here.

Reply:
The difference between the GPG & vaultkv  implementation is that I was asked not to store the metadata or the parameters in the secret file & instead get those from the inventory.
I couldn't find a method to get `target_name` and `inventory_path` once we have compiled the inventory. These parameters were required to load the vaultkv's `parameters` for revealing the secrets using the appropriate target's parameters.